File: VB/GridCommandToPopupMenuProject/Person.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GridCommandToPopupMenuProject
    Public Class Person
        Public Sub New(ByVal i As Integer)
            FirstName = String.Format("FirstName - {0}", i)
            LastName = String.Format("LastName - {0}", i)
        End Sub
        Public Property FirstName() As String
        Public Property LastName() As String
    End Class
End Namespace

File: CS/GridCommandToPopupMenuProject/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GridCommandToPopupMenuProject {
    public class Person {
        public Person(int i) {
            FirstName = string.Format("FirstName - {0}", i);
            LastName = string.Format("LastName - {0}", i);
        }
        public string FirstName { get; set; }
        public string LastName { get; set; }
    }
}
